source/_integrations/wiz.markdown

```diff
@@ -93,6 +93,27 @@ A power monitoring sensor is available on devices with the following hardware mo
 - ESP20_SHRGB_31R
 - ESP20_SHTW_31R
 
+Please note: the power monitoring sensor reports in Watts (W), rather than kiloWatts per hour (kWh). As a result, it is not compatible with the Energy Dashboard. There is currently a workaround for this:
```

Comment:
```suggestion
Please note: the power monitoring sensor reports in Watts (W), rather than kilowatts per hour (kWh). As a result, it is not compatible with the Energy Dashboard. There is currently a workaround for this:
```